src/program/snabb_lwaftr/setup.lua

```diff
@@ -47,10 +49,16 @@ function lwaftr_app(c, conf)
       prepend(postprocessing_apps_v6, "egress_filterv6")
    end
 
-   set_preprocessors(c, preprocessing_apps_v4, "lwaftr.v4")
-   set_preprocessors(c, preprocessing_apps_v6, "lwaftr.v6")
-   set_postprocessors(c, "lwaftr.v6", postprocessing_apps_v6)
-   set_postprocessors(c, "lwaftr.v4", postprocessing_apps_v4)
+   set_preprocessors(c, preprocessing_apps_v4, "icmpechov4.south")
+   config.link(c, "icmpechov4.north -> lwaftr.v4")
+   config.link(c, "lwaftr.v4 -> icmpechov4.north")
+
+   set_preprocessors(c, preprocessing_apps_v6, "icmpechov6.south")
+   config.link(c, "icmpechov6.north -> lwaftr.v6")
+   config.link(c, "lwaftr.v6 -> icmpechov6.north")
+
+   set_postprocessors(c, "icmpechov6.south", postprocessing_apps_v6)
+   set_postprocessors(c, "icmpechov4.south", postprocessing_apps_v4)
```

Comment:
Thanks for the suggestion. Just one note: the strings for `input` and `output` have to be swapped in the list of postprocessing applications (traffic is flowing in the opposite direction there).